### PR TITLE
Add data retrieval as custom call

### PR DESCRIPTION
**Pull request by qingweibinary**
Dont merge 1st, let's have test for this =)

### CODE REVIEW

**Comment by qingweibinary** [comment]
OK to merge if you find no issue

**Comment by qingweibinary** [comment]
Note: I published a beta version to test with nextgen ~

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "binary-live-api",
-  "version": "18.0.0",
+  "version": "18.0.1",
```
Comment (borisyankov) [inline package.json]: x.x.1 is for minor bug fixes
x.1.x is for small new features
1.x.x is for new features

In the end it doesn't matter too much, but still....

Comment (qingweibinary) [inline package.json]: so I should use 18.1.x ?

**Pull request merged**